frontend/src/app/modules/calendar/wp-calendar/wp-calendar.component.ts

```diff
@@ -14,18 +23,16 @@ import {DomSanitizer} from "@angular/platform-browser";
 import {WorkPackagesListChecksumService} from "core-components/wp-list/wp-list-checksum.service";
 import {OpTitleService} from "core-components/html/op-title.service";
 import dayGridPlugin from '@fullcalendar/daygrid';
-import {EventApi, EventInput} from '@fullcalendar/core';
-import {EventSourceError} from '@fullcalendar/core/structs/event-source';
+import {CalendarOptions, EventApi, EventInput} from '@fullcalendar/core';
 import {take} from 'rxjs/operators';
-import {ToolbarInput} from '@fullcalendar/core/types/input-types';
+import {ToolbarInput} from '@fullcalendar/common';
 import {ConfigurationService} from "core-app/modules/common/config/configuration.service";
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 import {SchemaCacheService} from "core-components/schemas/schema-cache.service";
 
 interface CalendarViewEvent {
   el:HTMLElement;
   event:EventApi;
-  jsEvent:MouseEvent;
 }
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/2d894dcd64d5257bf0f74084c726d7259a120e3a/frontend/src/app/modules/calendar/wp-calendar/wp-calendar.component.ts#L37"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>